connectors/es/client.py

```diff
@@ -50,17 +50,13 @@ def __init__(self, config):
         }
         logger.debug(f"Host is {self.host}")
 
-        if "username" in config:
```

Comment:
We could also add a warning log that we found both in the config and use the `api_key`, so someone could trace this behavior during support/debugging, if needed.

Reply:
warn log is a good idea